build-system/tasks/visual-diff/index.js

```diff
@@ -100,7 +100,7 @@ function setPercyBranch() {
  */
 function setPercyTargetCommit() {
   if (process.env.TRAVIS && !argv.master) {
-    process.env['PERCY_TARGET_COMMIT'] = gitBranchPointFromMaster();
+    process.env['PERCY_TARGET_COMMIT'] = gitMergeBaseTravisMaster();
```

Comment:
Aha. Were we getting spurious diffs as a result of this error?

Reply:
Yep, and for bundle-size too!